### ISSUE TITLE
Cannot find module '@drodil/backstage-plugin-qeta-backend'

### ISSUE THREAD

**Issue by conradoqg** [drodil/backstage-plugin-qeta#21]
After following the installation instructions, I'm getting the `Cannot find module '@drodil/backstage-plugin-qeta-backend'` message. Looking deeper it looks like de backend package is not built correctly, or something like that.

```
❯ tree ./node_modules/@drodil/backstage-plugin-qeta-backend/dist 
./node_modules/@drodil/backstage-plugin-qeta-backend/dist
├── main.05256229b1275fd3efba.hot-update.js
├── main.05256229b1275fd3efba.hot-update.json
├── main.20eaf34914d662a622d4.hot-update.js
├── main.20eaf34914d662a622d4.hot-update.json
├── main.3fa5c261f5c9dabeeef7.hot-update.js
├── main.3fa5c261f5c9dabeeef7.hot-update.json
├── main.459352cc2fed5ae6e0c9.hot-update.js
├── main.459352cc2fed5ae6e0c9.hot-update.json
├── main.4a4f904542185539c204.hot-update.js
├── main.4a4f904542185539c204.hot-update.json
├── main.57bb0d456721a5a5f281.hot-update.js
├── main.57bb0d456721a5a5f281.hot-update.json
├── main.68250dcf5acf7134a446.hot-update.js
├── main.68250dcf5acf7134a446.hot-update.json
├── main.764f7469059d6b292088.hot-update.js
├── main.764f7469059d6b292088.hot-update.json
├── main.869292e6bfcaee3b0d4d.hot-update.js
├── main.869292e6bfcaee3b0d4d.hot-update.json
├── main.91c4b7e4841b81c8ece5.hot-update.js
├── main.91c4b7e4841b81c8ece5.hot-update.json
├── main.c525b843d8944f3fb792.hot-update.js
├── main.c525b843d8944f3fb792.hot-update.json
├── main.d469fc9863c7624b8a8c.hot-update.js
├── main.d469fc9863c7624b8a8c.hot-update.json
└── main.js

0 directories, 25 files
```

By contrast, here is the 0.1.30 version folder structure.

```
❯ tree ./node_modules/@drodil/backstage-plugin-qeta-backend/dist
./node_modules/@drodil/backstage-plugin-qeta-backend/dist
├── index.cjs.js
├── index.cjs.js.map
└── index.d.ts

0 directories, 3 files
```

Am I missing something?

**Comment by padupe** [comment]
I came across the same error.

**Comment by padupe** [comment]
I also verified that it is something in version 1.0.0.
If you install, for example, version 0.1.30, the error does not occur, but TS complains about the identity property.

**Comment by drodil** [comment]
Hm. Weird. Probably the 1.0.0 is broken somehow. I try to publish a new version

**Comment by drodil** [comment]
@conradoqg @padupe I just released 1.0.1, could you try that out? I had to manually bump the version 1.0.0 so it might be that the build was somehow broken. If the new release works fine, I will deprecate the 1.0.0. If not, have to dig deeper what's going on.. I have used 0.1.30 in my own backstage instance and it worked fine, didn't have chance to test the 1.0.0.

**Comment by drodil** [comment]
Got time to verify myself, 1.0.1 seems to work fine. I will deprecate the 1.0.0. Please update to latest version. Closing this issue now.
